app/javascript/src/components/Common/NewPane.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Pane } from "neetoui";
+
+import NewContactForm from "components/Dashboard/Contacts/NewContactForm";
+import NewNoteForm from "components/Dashboard/Notes/NewNoteForm";
+
+export default function NewPane({
+  fetchNotes,
+  showPane,
+  setShowPane,
+  itemName,
+}) {
+  const onClose = () => setShowPane(false);
+  return (
+    <Pane title={`Add New ${itemName}`} isOpen={showPane} onClose={onClose}>
+      <div className="px-6">
+        {itemName === "Note" ? (
+          <NewNoteForm onClose={onClose} refetch={fetchNotes} />
+        ) : (
+          <NewContactForm onClose={onClose} refetch={fetchNotes} />
+        )}
+      </div>
+    </Pane>
+  );
+}
```

Comment:
Let's refrain from suffixing components or filenames with `Modal, Pane, Form, Page, etc`. Let's use semantic names instead. 
A filename `NewNote` indicates that the file/component handles the logic for creating a new note may it be using a Modal, Pane, or a new page with a form.